x-pack/platform/plugins/shared/maps/public/classes/sources/esql_source/esql_source.tsx

```diff
@@ -264,6 +264,10 @@ export class ESQLSource
 
     params.filter = buildEsQuery(undefined, query, filters, getEsQueryConfig(getUiSettings()));
 
+    if (requestMeta.projectRouting) {
```

Comment:
how does this interact with project routing defined in ESQL statement? Should param only be set of there is no project routing in ESQL statement